perl/imap/Cyrus/IndexFile.pm

```diff
@@ -2295,6 +2295,20 @@ sub _record_h2a {
   return \@array;
 }
 
+=item Cyrus::IndexFile->latest_version()
+
+=item $index->latest_version()
+
+Returns the latest (highest) version supported by Cyrus::IndexFile
+
+=cut
+
+sub latest_version {
+  my ($latest) = reverse sort { $a <=> $b } keys %$VersionFormats;
```

Comment:
Surprised you didn't sort `$b <=> $a` and drop the reverse.  I guess I have no objection other than finding it surprising?